application/features/user.feature

```diff
@@ -343,7 +344,7 @@ Feature: User
 #TODO: PUT with a change to an existing value on one of the unique fields should generate an error.
 #TODO: PUT with a change should update the last_changed data properly...as well as the field to be updated :-)
 
-
+#TODO: need to think about how to verify uuid comes back in calls.
```

Comment:
Do you think it would also be worth (eventually) adding a test to verify that the uuid cannot be changed by being sent in (in all scenarios)?